InvenTree/common/models.py

```diff
@@ -58,6 +58,13 @@ class InvenTreeSetting(models.Model):
             'description': _('String descriptor for the server instance'),
         },
 
+        'INVENTREE_INSTANCE_TITLE': {
+            'name': _('Use Instance Name'),
+            'description': _('Use the instance name in the Titel-Bar'),
```

Comment:
"Titel" should be "Title"

Reply:
oh sorry, my german is showing. Will fix ASAP